src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (int) Math.PI * radius * radius;
+    }
+
+    @Override
+    public String drawFigure() {
+        return "Figure: circle, area: " + calculateArea()
+                + " sq.units, radius: " + getRadius()

```

Comment:
```suggestion
                + " sq.units, radius: " + radius
```
recommendation: if you can access fields of this class directly - don't use getters
it is easier to read

Reply:
Yeah, I expected that this would be a thread, just wrote getters from the beginning and decided to keep it and see how it will be with you.

Reply:
Done!